pkg/awsManager/vpcVpnManager.go

```diff
@@ -199,13 +206,10 @@ func DeleteNatgateway(client clientpkg.Client, vpcID *string) error {
 			break
 		}
 	}
-
 	return nil
-
 }
 
-func DeleteNetworkLoadBalancer(client clientpkg.Client, vpcID *string) error {
-
+func deleteNetworkLoadBalancer(client clientpkg.Client, vpcID *string, logger logr.Logger) error {
```

Comment:
why was this unexported?

Reply:
because it is a helper method that isn't and shouldn't be used outside this file